src/types/entities/schelling.ts

```diff
@@ -96,12 +96,18 @@ export class SchellingGame {
 	public newBlock(block: BlockDetails): string {
 		const roundNo = SchellingGame.roundFromBlockNo(block.blockNo)
 		if (roundNo != this.currentRoundNo) {
+			// When the round changes, by defintion, no one is playing
+			this.players.forEachPair((overlay, player) => {
```

Comment:
Are we tracking the players that are playing in the current round? If so, should we not iterate over that list to reduce the length of this loop?

Reply:
Good catch!   I've modified that bit in a new fix: commit.   Feel free to squash the two into a single one.